<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectTogether</class>
 <widget class="QWidget" name="ConnectTogether">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Scientist ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Computer ID</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="displayConnections_tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>501</width>
     <height>201</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Scientist ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Computer ID</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>388</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="ScientistID_lineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="ComputerID_lineEdit_2"/>
    </item>
    <item>
     <widget class="QPushButton" name="AddConnection_pushButton">
      <property name="text">
       <string>Add Connection</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
